style.css

```diff
@@ -356,59 +367,1108 @@ span {
+  }
+
+  .project-des-d1 > ul > li:nth-child(1) {
+    width: 100px;
+    height: 30px;
+  }
+
+  .project-des-d1 > ul > li:nth-child(2) {
+    width: 120px;
+    height: 30px;
+  }
+
+  .project-des-d1 > ul > li:nth-child(3) {
+    width: 50px;
+    height: 30px;
+    margin-right: 1rem;
+  }
+
+  .project5-img-d {
+    background-image: url('./images/Rectangle\ 26.png');
     background-size: 100% 100%;
+    background-repeat: no-repeat;
+    height: 300px;
+  }
+
+  .aboutme-cards {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    list-style: none;
+    gap: 0;
+    padding: 0 0 0 0;
+  }
+
+  .aboutme-img-d {
+    background-image: url('./images/image_dots_with_teal.png');
+    background-repeat: no-repeat;
+    background-size: 65% 80%;
+    position: absolute;
+    left: 35%;
+    top: 21.5%;
+  }
+
+  .aboutme {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 4%;
+    justify-content: space-around;
+    gap: 1rem;
   }
 
   .aboutme-techs,
   .aboutme-framew,
-  .aboutme-skills {
+  .aboutme-skills,
+  .aboutme-img-d {
+    margin: 0 0 0 0;
     width: 100%;
-    margin-right: 1rem;
+    padding-top: 1rem;
+    position: relative;
+  }
+
+  .logo-corner1,
+  .logo-corner {
+    height: 90px;
+    left: 47.5%;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .logo-corner2 {
+    left: 45%;
+  }
+
+  .techs,
+  .framew,
+  .skills {
+    font-size: 14px;
+    margin: 0 0.2rem 0 0;
+  }
+
+  .aboutme-title {
+    font-size: 90px;
+  }
+
+  .aboutme-btn {
+    width: 254px;
+  }
+
+  .contact-form-d {
+    position: relative;
+    display: flex;
+    justify-content: space-evenly;
+    height: 950px;
+    background: #f9f9fa;
+  }
+
+  .contact-title-d {
     display: flex;
     flex-direction: column;
     align-items: center;
+    width: 100%;
   }
 
-  .aboutme-cards {
-    padding-right: 1rem;
+  .contact-card {
+    width: 70%;
+    padding: 0;
+    margin-top: 20%;
   }
 
-  .aboutme-p {
+  .contact {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 100%;
+    align-items: center;
+    gap: 0;
+  }
+
+  .contact-logo {
+    display: none;
+  }
+
+  .contact-logo1-d {
+    display: block;
+  }
+
+  .contact-logo2-d {
+    display: block;
+  }
+
+  .contact1-img {
+    max-width: 100%;
+    height: 26%;
+    position: absolute;
+    bottom: 0;
+    right: calc(80% - 1rem);
+    top: 0;
+  }
+
+  .contact-img {
+    max-width: 100%;
+    height: 26%;
+    position: absolute;
+    bottom: 0;
+    left: calc(72.5% - 1rem);
+    top: 74%;
+  }
+
+  .contact input[type="text"] {
+    width: 80%;
+    height: 52px;
+    background: #fff;
+    border-radius: 4px;
+  }
+
+  .contact textarea[name="your message"] {
+    width: 170%;
+  }
+
+  .input-d1,
+  .input-d2,
+  .input-d3 {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    width: 100%;
+  }
+
+  .input-d1 {
+    margin-left: 2rem;
+  }
+
+  .input-d2 {
+    margin-right: 2rem;
+  }
```

Comment:
- [ ] Kindly correct the size and the distance between the inputs to be similar to the Figma design 👇🏽 

| Figma design | Your design | 
|---|---|
| ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/afc21553-ebc2-43d3-be15-54b9930d112d) | ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/42bab56d-1b84-41b3-a80b-07a8427cdb85) |